src/sql/base/browser/ui/propertiesContainer/media/propertiesContainer.css

```diff
@@ -28,19 +28,23 @@ properties-container .columnLayout.property {
 	flex-direction: column;
 }
 
-properties-container .propertyName {
+properties-container .propertyName,
+properties-container .splitter {
 	opacity: 0.6;
 	font-size: 12px;
 	flex: 0 0 auto
 }
 
 .vs-dark properties-container .propertyName,
-.hc-black properties-container .propertyName {
+.hc-black properties-container .propertyName,
+.vs-dark properties-container .splitter,
+.hc-black properties-container .splitter {
 	opacity: 1;
 }
 
 properties-container .propertyValue {
 	font-size: 12px;
+	font-weight: 600;
```

Comment:
the latest dashboard mockups don't have the property values bolded. Is there a different mockup where they're bold?

Reply:
Where are those? I was going off of your image in https://github.com/microsoft/azuredatastudio/pull/10152

https://user-images.githubusercontent.com/31145923/80163470-1f484600-858b-11ea-8c3b-6e563a65d30c.png

(ok maybe those aren't actually bolded? They look a bit thicker than the other text but maybe that's just the opacity throwing me off)

Reply:
yeah might just look different in that screenshot. This is the css from figma:
![image](https://user-images.githubusercontent.com/31145923/80538901-58056800-895b-11ea-8c49-5ab16b646da2.png)